### ISSUE TITLE
Request for Library Name Change to Reduce Confusion in the Community

### ISSUE THREAD

**Issue by itleets** [MohamedRejeb/Ksoup#56]
Hi,

I recently came across two libraries named Ksoup on GitHub your library, `MohammedRejeb/Ksoup`, and `fleeksoft/ksoup`. While I appreciate your work, I believe that having two libraries with the same name is causing confusion among users. Most users expect `Ksoup` to be similar to Jsoup, but your library has a different focus and functionality.

I understand that your library was created first, but `fleeksoft/ksoup` is a direct port of Jsoup, with the same API design and feature set. Given this, I’d like to kindly request that you consider renaming your library to something different to avoid confusion in the community. Perhaps something like KsoupHTMLParser, KHTML, or KhtmlParser could work, but any name that distinguishes it would be helpful.

I’ve noticed others mentioning this confusion in your issue tracker and elsewhere, so I’m not alone in this concern. I understand that renaming a project can be challenging, but I believe it would be a great step toward helping the community.

Thank you for considering this!

**Comment by MohamedRejeb** [comment]
Hi,

Yes makes sense to me, I'm thinking of renaming the library. Hopefully, when I release 1.0 I will make that breaking change and rename it to something else.

Thanks!

**Comment by itleets** [comment]
Hi,

Thank you so much for understanding and considering the change! I think it will really help clear up any confusion for users. Looking forward to seeing your 1.0 release with the new name!

Thanks again, and best of luck with the project!

**Comment by FlykeSpice** [comment]
Yeah, I stumbled across this confusion recently when I got notification from r/Kotlin about the new release of a library named Ksoup developed by someone else.

https://www.reddit.com/r/Kotlin/comments/1fefb2g/comment/lmzj32o/
